demo/index.html

```diff
@@ -0,0 +1,175 @@
+<!--
+    @license
+    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<!doctype html>
+<html>
+<head>
+
+  <title>core-image</title>
+
+  <script src="../../webcomponentsjs/webcomponents.js"></script>
+
+  <link rel="import" href="../../polymer/polymer.html">
+  <link rel="import" href="../core-image.html">
+
+  <style>
+    .sized {
+      width: 200px;
+      height: 200px;
+    }
+    .gray {
+      background-color: lightgray;
+    }
+    .group {
+      display: inline-block;
+      vertical-align: top;
+    }
+
+    [hidden] {
+      display: none;
+    }
+  </style>
+
+</head>
+<body>
+  <template is="x-autobind">
+    <h3>Sizing: none (naturally sized)</h3>
+    <core-image src="./polymer.svg"></core-image>
+
+    <h3>Sizing: cover</h3>
+    <core-image class="sized" sizing="cover" src="./polymer.svg"></core-image>
+    <core-image class="sized" sizing="cover" src="./polymer.svg"></core-image>
+
+    <h3>Sizing: contain</h3>
+    <core-image class="sized gray" sizing="contain"  src="./polymer.svg"></core-image>
+    <core-image class="sized gray" sizing="contain"  src="./polymer.svg"></core-image>
+
+    <h3>Preload: none</h3>
+    <div class="group">
+      <div>No sizing</div>
+      <button on-click="preload" target="preload1a">Load image</button><br><br>
+      <core-image id="preload1a" class="sized gray" ></core-image>
+    </div>
+    <div class="group">
+      <div>Cover</div>
+      <button on-click="preload" target="preload1b">Load image</button><br><br>
+      <core-image id="preload1b" class="sized gray" sizing="cover" ></core-image>
+    </div>
+    <div class="group">
+      <div>Contain</div>
+      <button on-click="preload" target="preload1c">Load image</button><br><br>
+      <core-image id="preload1c" class="sized gray" sizing="contain" ></core-image>
+    </div>
+
+    <h3>Preload: color as placeholder</h3>
+    <div class="group">
+      <div>No sizing</div>
+      <button on-click="preload" target="preload2a">Load image</button>
+      <span hidden$="[[!loading2a]]">Loading...</span>
+      <br><br>
```

Comment:
(Not your code I know, but...) Don't use `<br>` tags for styling. Use margin/padding.